### PR TITLE
feat(LAMPy): new LAMPy integration

### PR DESCRIPTION
**Pull request by homelessbirds**

### CODE REVIEW

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+title: LAMPy stack monitoring integration
+tags:
+    - New Relic integrations
+    - LAMPy
+    - Linux
+    - Apache
+    - Python
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:8]: Adding mysql should also help
```
    - MySQL
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+metaDescription: Use New Relic browser monitoring to get a dashboard with metrics from your Flutter web application.
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:9]: metaDescription: Use the below New Relic's agents to get a dashboard with metrics for your LAMPy application.
1. Infrastructure agent
2. Apache agent
3. MySQL agent
4. Python agent

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+<figcaption>
+After setting up our LAMPy integration, we give you a dashboard for your LAMPy web app metrics. 
+</figcaption>
+
+## Install [#install]
+
+### Step 1: Install the infrastructure agent for Linux [#infra-install]
+
+To do this, follow [the infrastructure agent install steps](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) for the host containing your Flutter app.
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:32]: To do this, follow [the infrastructure agent install steps](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) for the host containing your LAMPy application.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+### Step 1: Install the MySQL agent [#mysql-install]
+
+Our LAMPy integration relies on the MySQL agent to work. To learn more and check requirements review our [MySQL docs](https://docs.newrelic.com/install/mysql/).
+
+1. From [one.newrelic.com](https://one.newrelic.com) click **Add data ** > Infrastructure & OS  > MySQL**.
+2.
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:39]: 2. Follow the instructions on the screen to install MySQL agent

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+
+## Find your data [#find-data]
+
+To get your LAMPy dashboard: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
+2. Click on **Dashboards**.
+3. In the search bar, type `LAMPy`. 
+4. The Flutter dashboard should appear. Click on it to install it.
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:88]: 4. The LAMPy dashboard should appear. Click on it to install it.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+Your LAMPy dashboard is considered a custom dashboard and can be found in the **Dashboards** UI. For docs on using and editing dashboards, see [our dashboard docs](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
+
+This integration reports data in the form of our [browser monitoring events](/docs/data-apis/understand-data/event-data/events-reported-browser-monitoring). If you installed the infrastructure agent, you'll also receive [infrastructure data](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data/#infrastructure-events). 
+
+Here's an example NRQL query checking the average window load time for the various browsers used by users on desktop computers:
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:94]: Below is the example NRQL query to check the Bytes Sent per Request.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+This integration reports data in the form of our [browser monitoring events](/docs/data-apis/understand-data/event-data/events-reported-browser-monitoring). If you installed the infrastructure agent, you'll also receive [infrastructure data](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data/#infrastructure-events). 
+
+Here's an example NRQL query checking the average window load time for the various browsers used by users on desktop computers: 
+
+```
+SELECT (average(`apache.server.net.bytesPerSecond`) 
+AVERAGE (`apache.server.net.requestsPerSecond`)) as 'Bytes sent per request' 
+FROM Metric 
+TIMESERIES auto
+```
```
Comment (pkosanam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:101]: Lines 96 to 101 has to be changed as below

```
SELECT (average(`apache.server.net.bytesPerSecond`) / average(`apache.server.net.requestsPerSecond`)) as 'Bytes sent per request' FROM Metric TIMESERIES auto
```

**Review by pkosanam — CHANGES_REQUESTED** [review]
Thank you for this PR..

Kindly use the below image as the screenshot 😃

https://github.com/newrelic/newrelic-quickstarts/blob/06f398a59a12f12da1152bce5c795deda5b22c50/dashboards/lampy/lampy01.png

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 8]
```diff
@@ -0,0 +1,105 @@
+---
+title: LAMPy stack monitoring integration
+tags:
+    - New Relic integrations
+    - LAMPy
+    - Linux
+    - Apache
+    - Python
+metaDescription: Use New Relics Linux, Apache, Python and MySql monitoring to get a dashboard with metrics from your LAMPy application.
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:9]:
```suggestion
metaDescription: Use New Relic Linux, Apache, Python and MySql monitoring to get a dashboard with metrics from your LAMPy application.
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+
+---
+
+import infrastructureLampyDashboard from 'images/infrastructure_screenshot-full_lampy-dashboard.png'
+
+Our LAMPy integration makes use of the Infrastructure agent, MySQL agent, Apache agent, and Python agent to provide you with a pre-built dashboard with your most important metrics like response time, CPU utilization, traffic, and login frequencies.
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:15]: I think in a lot of these docs we're calling many things 'agents' that we shouldn't be. We don't call our MySQL and Apache integrations 'agents' we just call them integrations. We reserve 'agent' for a few things, like the APM language agents, the infrastructure agent, browser agent, mobile agents. so I think we should reword this entire doc (and probably some docs we've already put out) to differentiate between agents and integrations. in this case, 'MySQL integration, Apache integration...'

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+
+### Step 1: Install the infrastructure agent [#infra-install]
+
+To do this, follow [the infrastructure agent install steps](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) for the host containing your LAMPy application.
+
+### Step 2: Install the MySQL agent [#mysql-install]
+
+Our LAMPy integration relies on the MySQL agent to work. To learn more and check requirements review our [MySQL docs](https://docs.newrelic.com/install/mysql/).
+
+1. From [one.newrelic.com](https://one.newrelic.com) click **Add data ** > Infrastructure & OS  > MySQL**.
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:38]:
```suggestion
1. From [one.newrelic.com](https://one.newrelic.com) click **Add data > Infrastructure & OS  > MySQL**.
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+2. Follow the instructions to install the MySQL agent.
+
+### Step 3: Install the Apache agent [#apache-install]
+
+Our LAMPy integration relies on the Apache agent to work. To learn more and check requirements review our [Apache docs](/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/).
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:43]: Just a note; some of the same issues here: 'integration', missing comma.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+
+1. From [one.newrelic.com](https://one.newrelic.com) click **Add data ** > Infrastructure & OS  > Apache**.
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:45]:
```suggestion
1. From [one.newrelic.com](https://one.newrelic.com) click **Add data > Infrastructure & OS  > Apache**.
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+2. Follow the instructions on the screen to install Apache agent.
+
+### Step 4: Install the Python agent [#python-install]
+
+Our LAMPy integration relies on the Python agent to work. To learn more and check requirements review our [Python docs](/docs/apm/agents/python-agent/getting-started/introduction-new-relic-python/).
+
+1. From [one.newrelic.com](https://one.newrelic.com) click **Add data ** > Application monitoring  > Python**.
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:52]:
```suggestion
1. From [one.newrelic.com](https://one.newrelic.com) click **Add data > Application monitoring  > Python**.
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 14]
```diff
@@ -0,0 +1,105 @@
...
+2. Name your application.
+3. Download the configuration file and place it in your application's root directory.
+4. Integrate python agent for Django web site which is running on Apache server
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:55]: Some of these are a little hard to parse.  #4 here is tough. Also 'Python'. Seems like it could use a readability pass as sometimes they're just sentence fragments/incomplete

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx [hunk 15]
```diff
@@ -0,0 +1,105 @@
...
+To get your LAMPy dashboard: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
+2. Click on **Dashboards**.
+3. In the search bar, type `LAMPy`. 
+4. The LAMPy dashboard should appear. Click on it to install it.
+
+Your LAMPy dashboard is considered a custom dashboard and can be found in the **Dashboards** UI. For docs on using and editing dashboards, see [our dashboard docs](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
+
+This integration reports data in the form of our [browser monitoring events](/docs/data-apis/understand-data/event-data/events-reported-browser-monitoring). If you installed the infrastructure agent, you'll also receive [infrastructure data](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-monitoring-data/#infrastructure-events).
```
Comment (zuluecho9) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:86]: this needs review. in this case, I think it will be reporting data from the infrastructure agent and the MySQL and Apache integrations, so maybe we just link to those docs? I'm not sure if it reports a smaller subset of that data but maybe see if the SMEs think it's okay to say 'This reports data from all these places' and just link to those docs.

**Review by zuluecho9 — DISMISSED** [review]

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx (hunk 15 above)
Comment (homelessbirds) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx:86]: @pkosanam what do you think about @zuluecho9's comment: "This reports data from all these places' and just link to those docs."

**Review by pkosanam — CHANGES_REQUESTED** [review]
Thank you for the PR.
Kindly address my comments and use the below dashboard image instead.

dashboard image: https://github.com/newrelic/newrelic-quickstarts/blob/06f398a59a12f12da1152bce5c795deda5b22c50/dashboards/lampy/lampy01.png

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx (hunk 15 above)
Comment (pkosanam) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx]: Hi @homelessbirds , As we have mentioned the configuration steps in the above paragraphs, I feel giving the documentation links here should be fine. 👍

Comment (homelessbirds) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx]: thanks @pkosanam !

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx (hunk 14 above)
Comment (homelessbirds) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/lampy-integration.mdx]: I think I fixed this!

**Review by pkosanam — APPROVED** [review]
looks good.. 👍

**Review by akristen — DISMISSED** [review]
Looks good! My only recommendation isn't a blocker: 

- I'd ditch the Install H2 and make those subsequent H3s their own categories, or even throw them into collapsers or a step component. Seeing the ordered lists batched into groups of 2 can impact readability. But the actual writing and content is good! Just a suggestion for a future iteration :)
